component-maps.yml

```diff
@@ -173,6 +173,13 @@ git:
     - mender-deviceconnect
     release_component: true
 
+  devicemonitor:
+    docker_image:
+    - devicemonitor
+    docker_container:
+    - mender-devicemonitor
+    release_component: false
```

Comment:
To not forget it, we should have a task with `fixVersion`2.9 to set this to `true`

Reply:
right! https://tracker.mender.io/browse/MEN-4678

Reply:
(the fix version to choose was "Mender 2.9", is that all right?)

Reply:
Not really, but 2.9.0 probably does not exist (yet). I'll fix.

Reply:
thank you (I was hesitant to add a new version as in I have never done it before)